### PR TITLE
Add Blockbench OBJ to minetest node mesh converter

### PR DESCRIPTION
**Pull request by Sneglium**
Adds a `sed` script that can be used to convert correctly-formatted OBJs exported from Blockbench into ones that can be used as Minetest node meshes.

### CODE REVIEW

**Comment by Sneglium** [comment]
Just realized I forgot to add a license, although I don't think the sed script is long enough to need one anyhow. Documentation text content can be CC0 or whatever works.

File: blockbench_to_minetest.sed [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Requires sed -r; use sed -i -r -f 'blockbench_to_minetest.sed' 'model.obj' for easiest results
+
+# Remove comments
+/^#\s.*$/d
```
Comment (appgurueu) [inline blockbench_to_minetest.sed:4]:
```suggestion
/^#.*$/d
```

I don't think comments need to start with a space

**Review by appgurueu — APPROVED** [review]
Looks good, having this script (and the associated documentation) is definitely better than not having it.

Please include a license (for both) though. I think a `SPDX-License-Identifier: <license>` comment is enough (in Markdown you can just use a HTML comment). Or just mention it in the Readme.

File: blockbench_to_minetest.sed (hunk 1 above)
Comment (Sneglium) [reply blockbench_to_minetest.sed:4]: True, that's redundant, I suppose I put it in there because blockbench's generated comments always have one

**Comment by Sneglium** [comment]
Removed the redundant `\s` and added a comment in both files marking them as CC0.

**Pull request merged**